src/theming/index.js

```diff
@@ -30,6 +29,12 @@ const colorObject = {
 
   "error": "var(--fallback-er,oklch(var(--er)/<alpha-value>))",
   "error-content": "var(--fallback-erc,oklch(var(--erc)/<alpha-value>))",
+
+  neutral: {
+    DEFAULT:  "var(--fallback-n,oklch(var(--n)/<alpha-value>))", 
+    ... colors.neutral,
```

Comment:
Instead of using `colors.neutral` from `tailwindcss/colors` please add the hardcoded `neutral` colors from `src/index.js`.

It's a workaround until daisyUI 5 release.  
After Tailwind 4/daisyUI 5 release, this whole thing won't be an issue anymore because Tailwind 4 uses CSS color-mix for opacity and all colors are CSS variables by default.